src/test/kotlin/com/exactpro/th2/http/client/ApplicationIntegrationTest.kt

```diff
@@ -125,24 +126,22 @@ class ApplicationIntegrationTest {
             }
         }.build())
 
-        assertNotNull(eventListener.poll(ofSeconds(2))).also {
-            expectThat(it) {
-                get { eventsList }.single().apply {
-                    get { name }.isEqualTo("Failed to handle transport message group")
-                    get { type }.isEqualTo("Error")
-                    get { status }.isEqualTo(EventStatus.FAILED)
-                    get { id }.apply {
-                        get { bookName }.isEqualTo(rootEventId.bookName)
-                        get { scope }.isEqualTo(rootEventId.scope)
-                    }
-                    get { parentId }.isEqualTo(rootEventId)
-                    get { attachedMessageIdsList }.isEmpty()
-                    get { body.toString(Charsets.UTF_8) }.isEqualTo(
-                        """
-                            [{"data":"java.net.ConnectException: Connection refused (Connection refused)","type":"message"}]
-                        """.trimIndent()
-                    )
+        expectThat(eventListener.poll(ofSeconds(2))).isNotNull().apply {
```

Comment:
Just a suggestion:

The Strikt API uses the builder pattern. This means you can chain method calls to do the assertion (without `apply` function).
For example,
```kotlin
expectThat(eventListener.poll(ofSeconds(2)))
 .isNotNull()
 .get { eventsList }
 .single().and {
    // other assertion
  }
```
In my opinion, it looks much clearer than the current approach